### PR TITLE
cmd/swarm/swarm-smoke: do not fail if a node does not respond to rpc

### PR DESCRIPTION
**Pull request by nonsense**
If one node of the deployment is offline, we are still interested in the distribution of chunks across the rest of the nodes.

### CODE REVIEW

File: cmd/swarm/swarm-smoke/upload_and_sync.go [hunk 1]
```diff
@@ -65,6 +65,14 @@ func uploadAndSyncCmd(ctx *cli.Context, tuid string) error {
 
 		return e
 	}
+
+	// trigger debug functionality on randomBytes even on successful runs
+	err := trackChunks(randomBytes[:])
+	if err != nil {
+		log.Error(err.Error())
```
Comment (holisticode) [inline cmd/swarm/swarm-smoke/upload_and_sync.go]: I wonder if we shouldn't return an error in this case. I know it's an edge case, as the actual test passed, but something else went wrong - unlikely but still possible, and it would show up as success in grafana. WDYT?

Comment (nonsense) [reply cmd/swarm/swarm-smoke/upload_and_sync.go]: We care mostly about the result of the smoke test. If for some reason the `debug` calls after that fail, we should not log the test as a failure, but still notice the error.
